notes/change-report-form-received.vbs

```diff
@@ -55,72 +56,94 @@ changelog_display
-  EditBox 60, 200, 305, 15, other_notes
-  EditBox 70, 220, 295, 15, verifs_requested
-  DropListBox 270, 240, 95, 20, "Select One:"+chr(9)+"will continue next month"+chr(9)+"will not continue next month", changes_continue
-  CheckBox 10, 245, 140, 10, "Check here to navigate to DAIL/WRIT", tikl_nav_check
-  EditBox 75, 260, 85, 15, worker_signature
+  EditBox 60, 185, 305, 15, actions_taken
+  EditBox 60, 205, 305, 15, other_notes
+  EditBox 70, 225, 295, 15, verifs_requested
+  CheckBox 10, 260, 140, 10, "Check here to navigate to DAIL/WRIT", tikl_nav_check
+  DropListBox 270, 255, 95, 20, "Select One:"+chr(9)+"will continue next month"+chr(9)+"will not continue next month", changes_continue
+  EditBox 75, 275, 85, 15, worker_signature
   ButtonGroup ButtonPressed
-	OkButton 260, 260, 50, 15
-	CancelButton 315, 260, 50, 15
+    OkButton 260, 275, 50, 15
+    CancelButton 315, 275, 50, 15
   Text 5, 10, 50, 10, "Case Number:"
   Text 110, 10, 50, 10, "Effective Date:"
   Text 210, 10, 110, 10, "Date Change Reported/Received:"
   GroupBox 5, 25, 365, 150, "Changes Reported:"
   Text 15, 40, 30, 10, "Address:"
   Text 15, 60, 35, 10, "HH Comp:"
-  Text 15, 80, 95, 10, "Assets (savings or property):"
-  Text 15, 100, 30, 10, "Vehicles:"
-  Text 15, 120, 30, 10, "Income:"
-  Text 15, 140, 25, 10, "Shelter:"
+  Text 15, 80, 30, 10, "Income:"
+  Text 15, 100, 25, 10, "Shelter:"
+  Text 15, 120, 95, 10, "Assets (savings or property):"
+  Text 15, 140, 30, 10, "Vehicles:"
   Text 15, 160, 20, 10, "Other:"
-  Text 10, 185, 45, 10, "Action Taken:"
-  Text 10, 205, 45, 10, "Other Notes:"
-  Text 10, 225, 60, 10, "Verifs Requested:"
-  Text 10, 265, 60, 10, "Worker Signature:"
-  Text 180, 245, 90, 10, "The changes client reports:"
+  Text 10, 190, 45, 10, "Action Taken:"
+  Text 10, 210, 45, 10, "Other Notes:"
+  Text 10, 230, 60, 10, "Verifs Requested:"
+  Text 10, 280, 60, 10, "Worker Signature:"
+  Text 180, 260, 90, 10, "The changes client reports:"
+  GroupBox 5, 175, 365, 75, "Actions"
 EndDialog
-'Shows dialog
+
+
+' Updated dialog to include field validation
 DO
 	DO
-		DO
-			DO
-    			Dialog Dialog1
-				cancel_confirmation
-				IF worker_signature = "" THEN MsgBox "You must sign your case note!"
-			LOOP UNTIL worker_signature <> ""
-			IF IsNumeric(MAXIS_case_number) = FALSE THEN MsgBox "You must type a valid numeric case number."
-		LOOP UNTIL IsNumeric(MAXIS_case_number) = TRUE
-		IF changes_continue = "Select One:" THEN MsgBox "You Must Select 'The changes client reports field'"
-	LOOP UNTIL changes_continue <> "Select One:"
-	call check_for_password(are_we_passworded_out)  'Adding functionality for MAXIS v.6 Passworded Out issue'
+		err_msg = ""
+		Dialog Dialog1
+		cancel_confirmation ' Asks user to confirm if they want to cancel
+
+		' Validate that MAXIS number is numeric and less than 8 digits long
+		CALL validate_MAXIS_case_number(err_msg, "* ")
+		' Validate that worker signature is not blank.
```

Comment:
Ok the only update on this pull request....
The mandatory field match information should follow the same kind of order like the tab order. As an example worker_signature should be the last err_msg shown to users in the message box.

Reply:
Makes sense! I updated the validation order accordingly